pkg/services/evaluation_report/evaluation_report_creator_test.go

```diff
@@ -0,0 +1,50 @@
+package evaluationreport
+
+import (
+	"github.com/gofrs/uuid"
+
+	"github.com/transcom/mymove/pkg/models"
+
+	"github.com/transcom/mymove/pkg/testdatagen"
+)
+
+func (suite *EvaluationReportSuite) TestEvaluationReportCreator() {
+	creator := NewEvaluationReportCreator()
+
+	suite.Run("Can create customer support report successfully", func() {
+		move := testdatagen.MakeDefaultMove(suite.DB())
+		shipment := testdatagen.MakeMTOShipment(suite.DB(), testdatagen.Assertions{
+			MTOShipment: models.MTOShipment{
+				MoveTaskOrderID: move.ID,
+			},
```

Comment:
I think what you've got should work fine, but will create and discard an extra `Move`.
If you pass the move through `Assertions.Move`, it should be a bit more efficient.

```suggestion
			Move: move,
```

Reply:
Great suggestion, that not only works better (no extra create/discard) but it also reads way more clearly. 💯 

Fixed ✅